lib/index.d.ts

```diff
@@ -1396,10 +1396,10 @@ declare namespace Stardog {
          * Adds a provider credential
          *
          * @param {Connection} conn the Stardog server connection
-         * @param {User} user the username and password to be stored
+         * @param {Credentials} credentials the credentials to be stored
```

Comment:
Need to also add `Credentials` type under `catalog` namespace (similar to how `DataSourceQuery` is under `dataSources` namespace.